src/App/App.xaml.cs

```diff
@@ -217,10 +217,9 @@ private async Task SleptAsync()
 
         private async void ResumedAsync()
```

Comment:
Let's take the opportunity to change this to async Task and use the FireAndForget method @fedemkr added in a previous PR, logging exceptions.